packages/ssr-web/config/environment.js

```diff
@@ -22,8 +22,9 @@ module.exports = function (environment) {
     locationType: 'auto',
     hubURL: process.env.HUB_URL,
     universalLinkDomain:
-      universalLinkHostnamesByTarget[process.env.DEPLOY_TARGET] ??
-      MERCHANT_PAYMENT_UNIVERSAL_LINK_STAGING_HOSTNAME,
+      universalLinkHostnamesByTarget[
+        process.env.DEPLOY_TARGET || process.env.SSR_WEB_ENVIRONMENT // front-end uses DEPLOY_TARGET, fastboot uses SSR_WEB_ENVIRONMENT
+      ] ?? MERCHANT_PAYMENT_UNIVERSAL_LINK_STAGING_HOSTNAME,
```

Comment:
I think `DEPLOY_TARGET` actually isn’t used at all in this application because that’s an Ember CLI Deploy thing.

Reply:
Is it worth keeping `DEPLOY_TARGET` and setting it in our CI for consistent language with `web-client`?

Reply:
I think we can remove `DEPLOY_TARGET`. Thanks @backspace for the tip, I think we can use `SSR_WEB_ENVIRONMENT` for both Sentry and app logic for links. Will adjust this